invenio/base/config.py

```diff
@@ -85,7 +85,6 @@
     'invenio.modules.archiver',
     'invenio.modules.cloudconnector',
     'invenio.modules.comments',
-    'invenio.modules.communities',  # remove with invenio/modules/communities
```

Comment:
keep this line to prevent loading of old module (you can just remove the comment)